### PR TITLE
Style: Fix visibility of state, city and services drop-downs

### LINKED ISSUE #1469: State, City and Services drop-downs selected value is not fully visible
In Essentials section State, City and Services dropdowns selected value is not fully visible in the Chrome browser. (My browser Version 81.0.4044.122 (Official Build) (64-bit))

OS: Windows 10
Display Resolution: 2160 x 3840 (4k Portrait mode)  or 3840 x 2160 (4k Landscape mode)

DropdownValuesAreNotReadable

### PR DESCRIPTION
**Pull request by Geetakokate**
**Description of PR**
My screen resolution is 1366x768. It is reproduced on 90% of my screen.
Fixed: State, City and Services drop-downs selected value is not fully visible issue.

**Relevant Issues**  
Fixes  #1469 

**Checklist**

- [ ] Properly formatted
- [ ] Tested on desktop
- [ ] Tested on phone

**Screenshots**
Before fix
covid19CSSissue

After fix
Screenshot from 2020-04-24 21-40-52

Add relevant screenshots here

### CODE REVIEW

**Comment by dharmesh-m** [comment]
@Geetakokate I checked current deployment in a 4k display. It is still reproducible. The Minimum required font size is 12.3px for 4k displays or we should adjust height of the dropdown if possible.

CurrentDeployment

MinimumRequired

File: src/components/Essentials/essentialsfiltersdesktop.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const useInputLabelStyles = makeStyles(() => ({
 const useMenuItemStyles = makeStyles(() => ({
   root: {
     fontFamily: 'archia',
-    fontSize: '11px !important',
+    fontSize: '11.5px !important',
     fontWeight: 600,
     textTransform: 'uppercase',
```
Comment (dharmesh-m) [inline src/components/Essentials/essentialsfiltersdesktop.js:41]: Adding extra property `minHeight: '14px',` solves the issue. You can revert fontSize changes. 
Make the same changes in file essentialsfiltersmobile.js as well for the mobile view.

File: src/components/Essentials/essentialsfiltersdesktop.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ const useInputLabelStyles = makeStyles(() => ({
 const useMenuItemStyles = makeStyles(() => ({
   root: {
     fontFamily: 'archia',
-    fontSize: '11px !important',
+    fontSize: '11.5px !important',
```
Comment (dharmesh-m) [inline src/components/Essentials/essentialsfiltersdesktop.js]: Revert this change

**Comment by dharmesh-m** [comment]
@shuklaayush @jeremyphilemon Can you add few reviewers in this PR?

File: src/components/Essentials/essentialsfiltersmobile.js [hunk 3]
```diff
@@ -25,6 +25,7 @@ const useInputLabelStyles = makeStyles(() => ({
   root: {
     fontFamily: 'archia',
     fontSize: '11px !important',
+    minHeight: '14px',
```
Comment (dharmesh-m) [inline src/components/Essentials/essentialsfiltersmobile.js:28]: This should be added in the object `useMenuItemStyles` just like in the `essentialsfiltersdesktop.js` file.
Also after doing additional testing in 4k display, it seems we require minHeight: '15px' 
Please do this for both desktop and mobile.

**Comment by shuklaayush** [comment]
Hi @Geetakokate, looks like this is obsolete now after the Essentials redesign. Thanks for the PR though! :)
